suite-native/module-onboarding-legacy/src/components/E2ESkipOnboardingButton.tsx

```diff
@@ -0,0 +1,3 @@
+export function E2ESkipOnboardingButton() {
```

Comment:
FYI, we use arrow functions for declaring functions/components.

Reply:
Updated.